.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+---
+label: 'needs review'
+---
+
+<!---
+Thank you for your pull request. Please provide below the name of the app, a short description 
+and a link to it's repository or homepage. Also, tell us why you think it's awesome!
+
+Make sure the PR adheres to our contribution guidelines:
+https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md
+-->
+
+#### New App Submission
+
+**App name:**
+
+**Repo or homepage link:**
+
+**Description:**
+
+**Why you think it's awesome:**
```

Comment:
End with a blank line so the user can click onto the last line directly.

Reply:
Done